src/Nox.ClientApi.Tests/Tests/Application/Messaging/IntegrationEventsTests.cs

```diff
@@ -137,7 +141,40 @@ public async Task Delete_Workplace_SendsIntegrationEvents()
             //Assert
             deleteResult.Should().NotBeNull();
 
-            (await MassTransitTestHarness.Published.Any<Nox.Messaging.NoxMessageRecord<WorkplaceDeleted>>()).Should().BeTrue();
+            // TODO: extend with event count so no additional events are sent, right now failing due to events from other test cases
+            //MassTransitTestHarness.Published.Count().Should().Be(1);
+            (await MassTransitTestHarness.Published.Any<NoxMessageRecord<WorkplaceDeleted>>()).Should().BeTrue();
+        }
+
+        [Fact]
+        public async Task Put_Country_SetsMessageValuesProperly()
```

Comment:
Can we have an additional test for a different environment (dev/test), by mocking the WebHostEnvironment?

Reply:
ok, new issue #959 for that.